### DISCUSSION TITLE
How is the ingestion password generated?

### DISCUSSION THREAD

**Discussion by gsakkis** [openobserve/openobserve#2773] [Q&A]
I noticed that the `ZO_ROOT_USER_PASSWORD` is different from the one shown in the ingestion page. For example I set `ZO_ROOT_USER_PASSWORD=123456789`, I can log in to the UI just fine but in the ingestion page the curl instructions show `curl -u <my-email>:bOm3doFTdgOhrZrd ...`. Likewise the authorization header for the OTEL collector corresponds to a base64 encoding of the other password (`bOm3doFTdgOhrZrd`), not the one I set.

**Comment by prabhatsharma** [comment]
`bOm3doFTdgOhrZrd` password can do only ingestion.
`ZO_ROOT_USER_PASSWORD` can call any other APIs too. 

The reason they are different is that, `bOm3doFTdgOhrZrd` will be exposed on many configuration files on many computers and you do not want to expose `ZO_ROOT_USER_PASSWORD`

**Reply by gsakkis** [reply]
Thanks, makes sense. So is this ingestion password randomly generated or it is somehow derived by the original username/password as described [here](https://openobserve.ai/docs/api/)?
